frontend-react/src/components/Crumbs.tsx

```diff
@@ -11,28 +16,49 @@ interface CrumbConfig {
 interface CrumbsProps {
     crumbList?: CrumbConfig[];
     noPadding?: boolean;
+    previousPage?: string;
 }
 
-const Crumbs = ({ crumbList, noPadding }: CrumbsProps) => {
-    if (crumbList) {
+const Crumbs = ({ crumbList, noPadding, previousPage }: CrumbsProps) => {
+    if (crumbList || previousPage) {
         return (
             <div className={!noPadding ? "grid-container" : ""}>
-                <BreadcrumbBar>
-                    {crumbList?.map((crumbConfig) => (
-                        <Breadcrumb
-                            key={`${crumbConfig.label}`}
-                            current={!crumbConfig?.path}
+                {previousPage !== undefined ? (
```

Comment:
Minor point and far, far, far from a blocker, but we can do `Boolean(previousPage)` to be a little more exclusive of all falsy types.  I realize TypeScript should help prevent this, but this'll help make it a little more robust in case we do eventually want to allow passing other falsy values (`null`|`false`|'""`, et cetera) down the line 🤷

Reply:
I do like the explicit expressiveness of it. Is there an eslint rule that we could use for this as a norm?

Reply:
Confusingly, typescript's Boolean is not typed correctly to allow called it as a function?

Reply:
I'm not sure about an ESLint rule (we can look around, though!), but that's very odd about TypeScript 🤔 🤔 🤔 

It seems okay locally for me!  What's the error that's coming up for you?

Reply:
Ok, now it's not complaining. Can't explain what happened there😕. Will commit the change